conf/ocsci/downstream_config.yaml

```diff
@@ -2,4 +2,3 @@
 ENV_DATA:
   ceph_image: 'quay.io/rhceph-dev/rhceph-4.0-rhel-8:latest'
   rook_image: 'quay.io/rhceph-dev/rook:latest'
-  cluster_namespace: 'rook-ceph'
```

Comment:
we can keep rook-ceph namespace because openshift-storage namespace leads to pod disappear

Reply:
done

Reply:
Restored rook-ceph namespace